atom/browser/native_window_mac.mm

```diff
@@ -682,6 +690,8 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
 
   options.Get(options::kTitleBarStyle, &title_bar_style_);
 
+  options.Get(options::kTabbingIdentifier, &tabbing_identifier_);
```

Comment:
Looks like `tabbing_identifier_` is only used in this method so it could be local variable here instead of an instance variable.